.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           php-version: ${{ matrix.php-version }}
           tools: composer:v2
-          coverage: none
+          coverage: xdebug
```

Comment:
@oliverklee Are you sure this is needed for CI build?

Reply:
I can try if it works without it, but I doubt it. Hold on.